examples/tls/main_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/VoltDB/voltdb-client-go/voltdbclient"
+)
+
+func TestMain(t *testing.T) {
+
+	conn, err := voltdbclient.OpenTLSConn("127.0.0.1", "foo.pem")
```

Comment:
I would just add param to OpenConn with ClientConfig and let OpenConn handle clear or tls connection.